app/views/layouts/_header.html.erb

```diff
@@ -17,19 +17,17 @@
       <li> <%= link_to "Cart", cart_path %> </li>
 
       <% if current_user %>
-        <% if current_admin? %>
+        <% if store_owner? %>
           <li><%= link_to "Dashboard", admin_store_path(current_user.store.slug) %></li>
-        <% else %>
-          <li><%= link_to "Profile", profile_path %></li>
         <% end %>
 
+        <li><%= link_to "Profile", profile_path %></li>
```

Comment:
I had a long list of stuff i found i just kept making notes for, i just happened to be in a place where that fix made sense.